lib/kernel/src/prim_tty.erl

```diff
@@ -525,6 +525,8 @@ writer_loop(TTY, WriterRef) ->
 -spec handle_request(state(), request()) -> {erlang:iovec(), state()}.
 handle_request(State = #state{ options = #{ tty := false } }, Request) ->
     case Request of
+        {putc_raw, Binary} ->
```

Comment:
Shouldn't this be added to the request type?

https://github.com/erlang/otp/blob/438a98a7e2dc8e39c0eeff91728d16b55581ea83/lib/kernel/src/prim_tty.erl#L166

Reply:
@frazze-jobb I added this to the request() type and force pushed the branch so we can get this into the testing over the weekend.